edb/ir/ast.py

```diff
@@ -157,9 +157,6 @@ class TypeRef(ImmutableBase):
     # Whether the union is specified by an exhaustive list of
     # types, and type inheritance should not be considered.
     union_is_exhaustive: bool = False
-    # If this is an intersection type, this would be a set of
-    # intersection elements.
-    intersection: typing.Optional[typing.FrozenSet[TypeRef]] = None
```

Comment:
It weirds me out a little if we aren't going to track that a type is an intersection in the TypeRef, even when it is one in the schema.

This might not actually matter though

Reply:
It's a bit odd, but the typeref doesn't exactly map onto types anyways